runt/src/main.rs

```diff
@@ -293,3 +296,15 @@ async fn attach(id: String) -> Result<(), Error> {
     }
     Ok(())
 }
+
+async fn start_repl(kernel_name: &String) -> Result<(), Error> {
```

Comment:
Since we're not providing a repl directly here (no input from user), I'd call this `start_runtime`